packages/eslint-plugin-x/package.json

```
@@ -44,6 +43,10 @@
     "eslint-plugin-vue": "~8.7.1",
     "eslint-plugin-vuejs-accessibility": "~2.0.0"
   },
+  "devDependencies": {
+    "eslint": "~8.30.0",
+    "prettier": "^2.0.5"
```

This should match root package.json prettier version.
```suggestion
    "prettier": "~2.3.2"
```